src/contactus/index.hbs

```diff
@@ -119,7 +119,38 @@
                     <section class="contact-map">
                         <div class="map-container">
                             <div class="col-md-12 map-box">
-                                <div class="map-canvas" data-address="Coding Blocks, Dwarka"
+                                <div class="map-canvas" data-latlong="28.5864285,77.0478732"
+                                     data-marker-title="Coding Blocks"
+                                     data-map-zoom="11" data-maps-api-key="AIzaSyCtSz4b43fPFC1kT90-x52A3ECJfBnHvN8"
+                                     data-map-style="[{&quot;stylers&quot;:[{&quot;hue&quot;:&quot;#dd0d0d&quot;}]},{&quot;featureType&quot;:&quot;road&quot;,&quot;elementType&quot;:&quot;labels&quot;,&quot;stylers&quot;:[{&quot;visibility&quot;:&quot;off&quot;}]},{&quot;featureType&quot;:&quot;road&quot;,&quot;elementType&quot;:&quot;geometry&quot;,&quot;stylers&quot;:[{&quot;lightness&quot;:100},{&quot;visibility&quot;:&quot;simplified&quot;}]}]"></div>
+                            </div>
+                        </div>
+                    </section>
+                </div>
+            </div>
+            <!-- Dehradun -->
+            <a id="dehradun" class="in-page-link"></a>
```

Comment:
Please note that contactus page still shows gurgaon as a centre. I have replaced it with Dehradun (as already present in /centres)